src/main/java/Deadline.java

```diff
@@ -0,0 +1,19 @@
+public class Deadline extends Todo {
+    private final String TYPE_SYMBOL = "[D]";
```

Comment:
Would it be better not to redeclare the TYPE_SYMBOL as Todo class already has this attribute? 

Perhaps can try to modify this variable in your Deadline constructor than redeclaring?